.devcontainer/rapids.Dockerfile

```
@@ -2,6 +2,7 @@
 
 ARG BASE
 ARG PYTHON_PACKAGE_MANAGER=conda
+ENV SCCACHE_IDLE_TIMEOUT=32768
```

Recommend lowering this to like 10-15 minutes.

The sccache server only reads AWS credentials at startup. So if the creds expire while the sccache server is still alive, it will still use them to read/write from S3.

To mitigate this, we rotate the credentials 2hrs before they're set to expire, which is hopefully enough time to finish any current builds and the server shuts down due to the idle timeout, then will read the new creds the next time it starts up again.

```suggestion
ENV SCCACHE_IDLE_TIMEOUT=900
```

---

Got it, thanks for the info.